alert.tf

```diff
@@ -15,9 +12,15 @@ data "azurerm_subscription" "current" {
   subscription_id = data.azurerm_client_config.current.subscription_id
 }
 
+module "log_analytics_workspace_id" {
+  source = "git@github.com:hmcts/terraform-module-log-analytics-workspace-id?ref=master"
+
+  environment = var.env
+}
+
 data "azurerm_log_analytics_workspace" "workspace" {
```

Comment:
This won't work for SDS as its in a different subscription, if you remove the data source and just use the ID output from the module on what is currently line 60